core/models.py

```diff
@@ -1,19 +1,42 @@
 from django.db import models
+from django.dispatch import receiver
+from django.db.models.signals import post_save
 
-# Create your models here.
-class Owner (models.Model):
-    full_name=models.CharFeild(max_length=50)
+#Model Manger
+class ProjectManager(models.Manager):
```

Comment:
Make use of Model Manager to filter objects or do something useful query to collect objects from the table.